padcli/jetconfig/jetconfig.go

```diff
@@ -61,13 +61,15 @@ type Config struct {
 // if the file doesn't exist
 func isPathFormatAConfigFile(path string) bool {
 	fi, err := os.Stat(path)
-	if err == nil && !fi.IsDir() {
-		return true
+	if err == nil {
+		// path already exists. If it's a dir it can't be a config file. If
+		// it's a file assume it's a config file.
+		return !fi.IsDir()
 	}
-	// Handle new configs:
-	// There's probably better more general ways to do this, but this if ok for
-	// now. Assume that if base contains a period it's a file.
-	return strings.ContainsRune(filepath.Base(path), '.')
+
+	// If path doesn't exist, always assume it's a config file. if more than the
+	// base of the path doesn't exist this will fail and that's ok.
```

Comment:
> if more than the
 	// base of the path doesn't exist
 	
what does this mean?

Reply:
if you provide a path like `/path/to/myconfig.yaml`  and `/path/to` exists then this works.

If you provide `/path/to/non-existent/myconfig.yaml` this will fail if `non-existent` doesn't exists. `myconfig.yaml` is the "base" of the path.

This is not a regression, we've never supported init for non-existent paths.